pkg/tls/renewer.go

```diff
@@ -96,7 +96,7 @@ func (c *certRenewer) RenewCA(ctx context.Context) error {
 	if err != nil && !apierrors.IsNotFound(err) {
 		return fmt.Errorf("failed to read CA (%w)", err)
 	}
-	now := time.Now()
+	now := time.Now().Add(c.renewBefore)
 	certs = removeExpiredCertificates(now, certs...)
 	if !allCertificatesExpired(now.Add(c.renewBefore), certs...) {
```

Comment:
since `c.renewBefore` has been added to `now` during declaration, this needs to be adjusted
```suggestion
	if !allCertificatesExpired(now, certs...) {
```

Reply:
made the changes. thanks.